simtools/simtel/simtel_histograms.py

```diff
@@ -79,17 +75,17 @@ def list_of_histograms(self):
         list:
             List of histograms.
         """
-
-        self._list_of_histograms = []
-        for file in self._histogram_files:
-            with EventIOFile(file) as f:
-                for o in yield_toplevel_of_type(f, Histograms):
-                    try:
-                        hists = o.parse()
-                    except Exception:  # pylint: disable=broad-except
-                        self._logger.warning(f"Problematic file {file}")
-                        continue
-                    self._list_of_histograms.append(hists)
+        if self._list_of_histograms is None:
+            self._list_of_histograms = []
+            for file in self._histogram_files:
+                with EventIOFile(file) as f:
+                    for o in yield_toplevel_of_type(f, Histograms):
+                        try:
+                            hists = o.parse()
+                        except Exception:  # pylint: disable=broad-except
```

Comment:
Do you know what type of Exceptions are these? Maybe add a test as well when this Exception is triggered.

Reply:
`except Exception` catches the exceptions dealt with in pyeventio but not `BaseException` as I understood from [here](https://stackoverflow.com/questions/18982610/difference-between-except-and-except-exception-as-e). 

The `# pylint: disable=broad-except` lets us not test this line and get away with it, but indeed it is at least important to pass on the information, hence I added the message to the logger now.